ui/app/src/state/actions/content.ts

```diff
@@ -28,59 +28,40 @@ export const fetchQuickCreateListComplete = /*#__PURE__*/ createAction<QuickCrea
 export const fetchQuickCreateListFailed = /*#__PURE__*/ createAction('FETCH_QUICK_CREATE_LIST_FAILED');
 // endregion
 
-// region Detailed Item
+// region Content Item
 
-export const fetchDetailedItem = /*#__PURE__*/ createAction<{ path: string }>('FETCH_DETAILED_ITEM');
+export const fetchContentItem = /*#__PURE__*/ createAction<{ path: string }>('FETCH_CONTENT_ITEM');
 
-export const reloadDetailedItem = /*#__PURE__*/ createAction<{ path: string }>('RELOAD_DETAILED_ITEM');
+export const reloadContentItem = /*#__PURE__*/ createAction<{ path: string }>('RELOAD_CONTENT_ITEM');
 
-export const completeDetailedItem = /*#__PURE__*/ createAction<{ path: string; force?: boolean }>(
-	'COMPLETE_DETAILED_ITEM'
+export const completeContentItem = /*#__PURE__*/ createAction<{ path: string; force?: boolean }>(
+	'COMPLETE_CONTENT_ITEM'
 );
 
-export const fetchDetailedItemComplete = /*#__PURE__*/ createAction<DetailedItem>('FETCH_DETAILED_ITEM_COMPLETE');
+export const fetchContentItemComplete = /*#__PURE__*/ createAction<ContentItem>('FETCH_CONTENT_ITEM_COMPLETE');
 
-export const fetchDetailedItemFailed = /*#__PURE__*/ createAction<AjaxError>('FETCH_DETAILED_ITEM_FAILED');
-
-export const fetchDetailedItems = /*#__PURE__*/ createAction<{ paths: string[] }>('COMPLETE_DETAILED_ITEMS');
-
-export const fetchDetailedItemsComplete = /*#__PURE__*/ createAction<{ items: DetailedItem[] }>(
-	'FETCH_DETAILED_ITEMS_COMPLETE'
-);
-
-export const fetchDetailedItemsFailed = /*#__PURE__*/ createAction<AjaxError>('FETCH_DETAILED_ITEMS_FAILED');
+export const fetchContentItemFailed = /*#__PURE__*/ createAction<AjaxError>('FETCH_CONTENT_ITEM_FAILED');
 
 // endregion
 
-// region Sandbox Item
-
-export type FetchSandboxItemPayload = { path: string };
-
-export type FetchSandboxItemsPayload = { paths: string[] };
-
-export const fetchSandboxItem = /*#__PURE__*/ createAction<FetchSandboxItemPayload>('FETCH_SANDBOX_ITEM');
-
-export const fetchSandboxItems = /*#__PURE__*/ createAction<FetchSandboxItemsPayload>('FETCH_SANDBOX_ITEMS');
-
-export type FetchSandboxItemCompletePayload = { item: SandboxItem };
+// region Content Items
 
-export type FetchSandboxItemsCompletePayload = { items: SandboxItem[] };
+export type FetchContentItemsPayload = { paths: string[] };
```

Comment:
Is this right? Why did it change from SandboxItem[] to string[]?

Reply:
There's something confusing in github's diff. FetchContentItemsPayload used to be FetchSandboxItemsPayload, they have the same payload type. FetchSandboxItemsCompletePayload is now FetchContentItemsCompletePayload, also have the same payload type.